gogs/gogs
# [F] Standardize HTTP status codes

## Standardize HTTP status codes

1. Improved Flexibility: Allows for dynamic status code determination based on specific error contexts.
2. Enhanced Accuracy: Prevents the misuse of 200 OK in cases where an error status is more appropriate.

Link to the issue: https://github.com/gogs/gogs/issues/7842

## Checklist


## Test plan

1. Clone repo and get tools
2. Test the project using go test ./...
4. Build the project using go build -o gogs
5. Run server, and check Admin Dashboard page
